packages/testing/src/deployment.cairo

```diff
@@ -1,11 +1,16 @@
 use crate::panic_data_to_byte_array;
-use snforge_std::{ContractClass, ContractClassTrait};
+use snforge_std::{ContractClass, ContractClassTrait, DeclareResult};
 use starknet::ContractAddress;
 
 /// Declares a contract with a `snforge` `declare` call and unwraps the result.
 pub fn declare_class(contract_name: ByteArray) -> ContractClass {
```

Comment:
Since this change affects the `declare_and_deploy` and `declare_and_deploy_at` functions behavior, I think it is worth adding a note to those stating that the function will skip declaration if the contract is already declared. A note on this function could be helpful as well.

Reply:
Good catch! I just update the inline  docs and the antora docs. Let me know what you think.